src/libs/Navigation/FreezeWrapper.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState, useRef} from 'react';
+import lodashFindIndex from 'lodash/findIndex';
+import PropTypes from 'prop-types';
+import {useIsFocused, useNavigation, useRoute} from '@react-navigation/native';
+import {Freeze} from 'react-freeze';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
```

Comment:
Also, why is `withWindowDimensions` used here? I don't see any uses of the props. Is it just so it re-renders on resize? Or could we remove it?

Reply:
Good catch! I think that may be some leftovers. I am going to remove it and test if everything is working correctly, but it should be okay